Dockerfile

```diff
@@ -225,21 +225,21 @@ RUN apk --update --no-cache add \
     nginx-mod-http-dav-ext \
     nginx-mod-http-geoip2 \
     openssl \
-    php83 \
-    php83-bcmath \
-    php83-ctype \
-    php83-curl \
-    php83-dom \
-    php83-fileinfo \
-    php83-fpm \
-    php83-mbstring \
-    php83-openssl \
-    php83-phar \
-    php83-posix \
-    php83-session \
-    php83-sockets \
-    php83-xml \
-    php83-zip \
+    php84 \
```

Comment:
I think we need to create a symlink for `/usr/bin/php` as php84 is not the default: https://pkgs.alpinelinux.org/contents?file=&path=&name=php84&branch=v3.22&repo=community&arch=x86_64

Compare to php83: https://pkgs.alpinelinux.org/contents?file=&path=&name=php83&branch=v3.22&repo=community&arch=x86_64

Reply:
Alright, I got it working. It was `php-config` that required the symlink.
https://pkgs.alpinelinux.org/contents?file=php-config84&path=&name=&branch=v3.22&repo=community&arch=x86_64